docker-compose-prod.yaml

```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: .
       dockerfile: ./Containerfile.cuda
-    hostname: "${LLAMA_CPP_HOST}"
+    hostname: llama-cpp-server
```

Comment:
Please add note that this has to match configuration in the config file. It is obvious, but previously we didn't have to care about that.

Reply:
will add it